src/containers/Storage/PDisk/PDisk.scss

```diff
@@ -21,7 +21,7 @@
     }
 
     &__vdisks-item {
-        flex-basis: 5px;
+        flex-basis: 3px;
```

Comment:
Allow `PDisk` to have more compact `VDisks` on it. It's needed to make `StorageNodes` to have the same row height, allowing switching from `variable` RDT rendering to `uniform`